src/main.h

```diff
@@ -288,13 +288,13 @@ void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
 bool AcceptTxBaseToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransactionBase &txBase, bool fLimitFree,
-                        bool* pfMissingInputs, bool fRejectAbsurdFee=false, bool disconnecting = false);
+                        bool* pfMissingInputs,  bool disconnecting, bool fRejectAbsurdFee=false);
 
 bool AcceptTxToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fRejectAbsurdFee=false, bool disconnecting = false);
+                        bool* pfMissingInputs, bool disconnecting, bool fRejectAbsurdFee=false, bool fVerifyBwtRequests = true);
```

Comment:
accept to mempool must always verifyBwtRequests. Remove fVerifyBwtRequests  param

Reply:
Done in mbtr_introduction_final_fixes